.circleci/config.yml

```diff
@@ -71,18 +71,7 @@ commands:
       - run:
           name: Tag build
           command: echo "<< parameters.target_branch >> $(date -u +'%Y-%m-%dT%H:%M:%SZ')" > dist/version
-      - run:
-          name: Install and configure dependencies
-          command: |
-            sudo npm install -g gh-pages@2.0.1
-            git config user.email "ci-build@binary.com"
-            git config user.name "ci-build"
-      - add_ssh_keys:
-          fingerprints:
-            - "01:67:4a:6d:26:9c:70:c4:1a:60:91:88:d9:dd:f0:83"
-      - run:
-          name: Deploy docs to gh-pages branch
-          command: gh-pages -d dist --branch << parameters.target_branch >> --message '[skip ci]'
+
```

Comment:
lets remove the empty line at line 74 and 179. PR looks completely fine, its just an optional requirement

Reply:
Removed